resources/functionMap_php80delta.php

```diff
@@ -45,7 +45,7 @@
 		'get_resource_id' => ['int', 'res'=>'resource'],
 		'gmdate' => ['string', 'format'=>'string', 'timestamp='=>'int'],
 		'gmmktime' => ['int|false', 'hour'=>'int', 'minute='=>'int', 'second='=>'int', 'month='=>'int', 'day='=>'int', 'year='=>'int'],
-		'hash_hkdf' => ['string', 'algo'=>'string', 'ikm'=>'string', 'length='=>'int', 'info='=>'string', 'salt='=>'string'],
+		'hash_hmac' => ['non-empty-string', 'algo'=>'string', 'data'=>'string', 'key'=>'string', 'binary='=>'bool'],
```

Comment:
did you intentionally replace `hash_hkdf` with `hash_hmac`?

Reply:
No, sorry :D